client/elements/addons/sc-static-page.html

```diff
@@ -0,0 +1,56 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<!--
+  Static page scrolling
+-->
+<dom-module id="sc-static-page">
+  <template>
+  </template>
+
+  <script>
+      class SCStaticPage extends Polymer.mixinBehaviors([Polymer.AppLocalizeBehavior], ReduxMixin(Polymer.Element)) {
+
+          static get properties() {
+              return {
+                  currentId: {
+                      type: String,
+                      value: ''
+                  }
+              }
+          }
+
+          connectedCallback() {
+            super.connectedCallback();
+          }
+
+          ready() {
+              super.ready();
+              this.shadowRoot.querySelector('main').addEventListener('click', () => {
```

Comment:
This will fail with an error if there is no `<main>` element in the subclass. It should either be wrapped in a `try/catch` block or an `if this.shadowRoot.querySelector('main')` condition.